eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -31,16 +43,19 @@ foreach ($branch in $branches)
   }
 
   $openPullRequests = $pullRequests | ? { $_.State -eq "open" }
-
-  if ($openPullRequests.Count -eq 0)
+  if ($openPullRequests.Count -gt 0)
   {
-    LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has no associated open Pull Request. Deleting Branch"
-    try{
-      Remove-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref ($branch.Remove(0,5)) -AuthToken $AuthToken
-    }
-    catch {
-      LogError "Remove-GitHubSourceReferences failed with exception:`n$_"
-      exit 1
-    }
+    LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has open pull Requests. Skipping"
+    LogDebug $openPullRequests.url
```

Comment:
Does this correctly output if the pull requests list is more then one?

Reply:
Yes that works